lib/src/json_schema/validator.dart

```diff
@@ -304,43 +335,45 @@ class Validator {
       final minContains = schema.minContains;
       final containsItems = instance.data.where((item) => Validator(schema.contains).validate(item)).toList();
       if (minContains is int && containsItems.length < minContains) {
-        _err('minContains violated: $instance', instance.path, schema.path);
+        warnOrErr('minContains violated: $instance', instance.path, schema.path);
       }
       if (maxContains is int && containsItems.length > maxContains) {
-        _err('maxContains violated: $instance', instance.path, schema.path);
+        warnOrErr('maxContains violated: $instance', instance.path, schema.path);
       }
       if (containsItems.length == 0 && !(minContains is int && minContains == 0)) {
-        _err('contains violated: $instance', instance.path, schema.path);
+        warnOrErr('contains violated: $instance', instance.path, schema.path);
       }
     }
   }
 
   void _validateAllOf(JsonSchema schema, Instance instance) {
+    final warnOrErr = _errFuncForVocabulary(SupportedVocabularies.APPLICATOR);
     if (!schema.allOf.every((s) => Validator(s).validate(instance))) {
-      _err('${schema.path}: allOf violated ${instance}', instance.path, schema.path + '/allOf');
+      warnOrErr('${schema.path}: allOf violated ${instance}', instance.path, schema.path + '/allOf');
     }
   }
 
   void _validateAnyOf(JsonSchema schema, Instance instance) {
+    final warnOrErr = _errFuncForVocabulary(SupportedVocabularies.APPLICATOR);
     if (!schema.anyOf.any((s) => Validator(s).validate(instance))) {
-      // TODO: deal with /anyOf
```

Comment:
I _think_ these TODO comments were stale? If not, let me know.

Reply:
I think these comments are about not passing back error messages from the validations.

Reply:
Oh! Then I'll put them back in a more helpful form.